packages/@divvi/mobile/src/earn/EarnWithdrawConfirmationScreen.tsx

```diff
@@ -40,29 +46,69 @@ const TAG = 'earn/EarnWithdrawConfirmationScreen'
 
 type Props = NativeStackScreenProps<StackParamList, Screens.EarnWithdrawConfirmationScreen>
 
+function getTotalWithdrawAmountsPerToken<
```

Comment:
just to confirm this is something specific to the new bottom sheet on the confirmation screen right? We don't have anything like this in the existing flow right?